code/controllers/subsystem/persistence.dm

```diff
@@ -188,27 +194,73 @@ SUBSYSTEM_DEF(persistence)
 
 	return json
 
-
-/datum/controller/subsystem/persistence/proc/LoadTrophies()
+/// Loads the trophies from the source file, and places a few in trophy display cases.
+/datum/controller/subsystem/persistence/proc/load_trophies()
+	var/list/raw_saved_trophies = list()
 	if(fexists("data/npc_saves/TrophyItems.sav")) //legacy compatability to convert old format to new
 		var/savefile/S = new /savefile("data/npc_saves/TrophyItems.sav")
 		var/saved_json
 		S >> saved_json
 		if(!saved_json)
 			return
-		saved_trophies = json_decode(saved_json)
+		raw_saved_trophies = json_decode(saved_json)
 		fdel("data/npc_saves/TrophyItems.sav")
-	else
+	else if(fexists("data/npc_saves/TrophyItems.json"))
 		var/json_file = file("data/npc_saves/TrophyItems.json")
 		if(!fexists(json_file))
 			return
 		var/list/json = json_decode(file2text(json_file))
 		if(!json)
 			return
-		saved_trophies = json["data"]
-	SetUpTrophies(saved_trophies.Copy())
-
-/datum/controller/subsystem/persistence/proc/LoadRecentMaps()
+		raw_saved_trophies = json["data"]
+		fdel("data/npc_saves/TrophyItems.json")
+	else
+		var/json_file = file("data/trophy_items.json")
+		if(!fexists(json_file))
+			return
+		var/list/json = json_decode(file2text(json_file))
+		if(!json)
+			return
+		raw_saved_trophies = json["data"]
+
+	for(var/raw_json in raw_saved_trophies)
+		var/datum/trophy_data/parsed_trophy_data = new
+		parsed_trophy_data.load_from_json(raw_json)
+		saved_trophies += parsed_trophy_data
+
+	set_up_trophies()
+
+///trophy data datum, for admin manipulation
+/datum/trophy_data
+	///path of the item the trophy will try to mimic, null if path_string is invalid
+	var/path
+	///the message that appears under the item
+	var/message
+	///the key of the one who placed the item in the trophy case
+	var/placer_key
+
+/datum/trophy_data/proc/load_from_json(list/json_data)
+	path = json_data["path"]
+	message = json_data["message"]
+	placer_key = json_data["placer_key"]
+
+/datum/trophy_data/proc/to_json()
+	var/list/new_data = list()
+	new_data["path"] = path
+	new_data["message"] = message
+	new_data["placer_key"] = placer_key
+	new_data["is_valid"] = text2path(path)
```

Comment:
should probably be TRUE/FALSE instead of null

Reply:
Done.